apps_validation/json_schema_utils.py

```diff
@@ -1,3 +1,21 @@
+APP_ITEM_JSON_SCHEMA = {
+    'type': 'object',
+    'properties': {
+        'categories': {
```

Comment:
`categories/tags` are a required field and we should be enforcing that they are present